app/uk/gov/hmrc/nisp/views/excluded_sp.scala.html

```diff
@@ -50,22 +50,26 @@ <h2 class="heading-medium"> @Html(Messages("nisp.excluded.willReach", Dates.form
         <div class="panel-indent">
             @if(spExclusions.contains(CustomerTooOld) || spExclusions.contains(PostStatePensionAge)) {
                     @exclusions.sp.postStatePensionAge()
-            } else { 
-
-                @if(spExclusions.contains(AmountDissonance)) {
-                    @exclusions.sp.amountDissonance()
+            } else {
+                @if(spExclusions.contains(ManualCorrespondenceIndicator)) {
+                    @exclusions.sp.isleOfMan()
```

Comment:
I don't think we need an MCI if statement here. And it shouldn't go to the IOM page...